internal/cmd/schema-registry/command.go

```diff
@@ -46,9 +46,20 @@ func New(cfg *v1.Config, prerunner pcmd.PreRunner, srClient *srsdk.APIClient) *c
 func addCompatibilityFlag(cmd *cobra.Command) {
```

Comment:
This function was created only to add the `--compatibility` flag. Can we create similar functions for all 5 of the new flags if they're used by more than one command?

Reply:
Do you mean to break it down into 5 functions? Since we enhance our API so all 6 parameters are updatable. Would it be better to just rename this function?

Reply:
Yes please! It's best to keep them separate so we can be more explicit in the command definition function.